Sources/AppStoreConnectCLI/Services/Operations/GetBetaTestersInfoOperation.swift

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct GetBetaTestersInfoOperation: APIOperation {
+
+    private enum GetTestersError: LocalizedError {
+        case betaTesterNotFound(String)
+
+        var errorDescription: String? {
+            switch self {
+            case .betaTesterNotFound(let email):
+                return "Beta tester with provided email '\(email)' doesn't exist."
+            }
+        }
+    }
+
+    private var endpoint: APIEndpoint<BetaTestersResponse> {
+        var limits: [ListBetaTesters.Limit] = []
+
+        if let limitApps = options.limitApps {
+            limits.append(.apps(limitApps))
+        }
+
+        if let limitBuilds = options.limitBuilds {
+            limits.append(.builds(limitBuilds))
+        }
+
+        if let limitBetaGroups = options.limitBetaGroups {
+            limits.append(.betaGroups(limitBetaGroups))
+        }
+
+        return APIEndpoint.betaTesters(
+            filter: [.email([options.email])],
+            include: [.betaGroups, .apps],
+            limit: limits
+        )
+    }
+
+    let options: GetBetaTesterInfoOptions
```

Comment:
You can definitely bring the Options type inside the operation and do away with the `GetBetaTesterInfoOptions` file

Reply:
that will cause a pretty long function name like, eg 
```
getBetaTester(
    email: String
    limitApps: Int?
    limitBuilds: Int?
    limitBetaGroups: Int?
)
```
especially for some operations with a long list of filters 🤔 🤔 🤔 , what do you think

Reply:
I'm thinking that commands don't need to really know about operations, it does result in more function params, I think that's okay. But for now if you want you can still use it like the example below while we talk about it.

E.g. `getBetaTester(options: GetBetaTesterInfoOperation.Options)`